[+/-] add btn for copy murr

das-dev opened the pull request:
- add absurl templatetag for generate absurl outer templates;
- register murr as app for access to project-wide templatetags;
- add .input_default and .btn_default base styles;
- add FontAwesome as font and custom FontAwesome css-rules;
- rename murr-list.css to murr-card.css;
- refactor murr-card html and styles;
- drop .action class, use .is_clickable instead;

Kuzyashin commented on murr/templatetags/absurl.py:
```diff
@@ -0,0 +1,20 @@
+class AbsoluteURLNode(URLNode):
+    def render(self, context):
+        path = super(AbsoluteURLNode, self).render(context)
+        domain = "http://127.0.0.1:8000"
```
Может сделать так? : 
`domain = context.request.build_absolute_uri('/')[:-1]`

das-dev replied:
Если это рабочее решение, то очень здорово: хотелось бы уйти от хардкода хоста.

das-dev replied:
done

Kuzyashin commented on static/css/murr_card.css:
```diff
@@ -33,69 +35,74 @@
+.murr-card__toggle {
+    position: absolute;
+    z-index: 15;
+    top: .2rem;
```
Предлагаю тут поставить .75rem  для top и right
http://prntscr.com/o5d6il

das-dev replied:
Не сравнивал пока с тем что есть, но у тебя на скрине выглядит хорошо

das-dev replied:
done

Kuzyashin commented on static/js/murr_list.js:
```diff
@@ -52,10 +45,13 @@
+                    } else if ($(event.target).hasClass('js-copy-url')) {
+                        let $murrLink = $('.js-murr-link', $this);
+                        $murrLink.focus().select();
+                        document.execCommand('copy');
```
На iOS c этим проблемы. У меня копировать не хочет. Гугл говорит что надо подобные конструкции строить https://toster.ru/q/543544

das-dev replied:
Ок, надо будет изучить вопрос.

das-dev replied:
@Kuzyashin чекни

Kuzyashin reviewed (commented):
Review

Commit:
[+/-] add btn for copy murr

- add absurl templatetag for generate absurl outer templates;
- register murr as app for access to project-wide templatetags;
- add .input_default and .btn_default base styles;
- add FontAwesome as font and custom FontAwesome css-rules;
- rename murr-list.css to murr-card.css;
- refactor murr-card html and styles;
- drop .action class, use .is_clickable instead;

Commit:
[+/-] fixes for copy murr

- replace hardcoded host with context.request.build_absolute_uri('/');
- fixed murr menu btn positioning;

Commit:
[+/-] fixes for copy murr

- rename absurl template tag to fullurl;
- extract copy link code to distinct method;
- implement selection via HTMLInputElement.setSelectionRange();
- add to copy-link input readonly attribute;

Commit:
[+/-] minor fixes for copy murr

Commit:
Merge branch 'develop' into copy-murr

# Conflicts:
#	static/css/murr_card.css
#	templates/MurrCard/includes/_murr_list_card.html
